js/ui/color_box/color_view.js

```diff
@@ -374,9 +382,10 @@ const ColorView = Editor.inherit({
             boundOffset: (function() {
                 return -this._paletteHandleHeight / 2;
             }).bind(this),
-            onDragMove: (function() {
+            onDragMove: (function(args) {
```

Comment:
```suggestion
            onDragMove: ({ event }) => {
```

and remove `.bind(this)` below